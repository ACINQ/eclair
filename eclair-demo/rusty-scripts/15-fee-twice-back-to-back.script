# Test committing fees before receiving previous revocation.
A:nocommitwait
A:feechange
A:commit
A:feechange
A:commit
echo ***A after two commits with fee changes***
A:dump
B:recvfeechange
echo ***B after receiving fee change***
B:dump
B:recvcommit
echo ***B after receiving first commit***
B:dump
B:recvfeechange
B:recvcommit
echo ***B after receiving second commit***
B:dump
A:recvrevoke
A:recvrevoke
B:commit
A:recvcommit
B:recvrevoke
echo ***B after committing***
B:dump
checksync
