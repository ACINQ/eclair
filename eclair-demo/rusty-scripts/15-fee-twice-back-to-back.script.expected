***A after two commits with fee changes***
LOCAL COMMITS:
 Commit 0:
  Offered htlcs:
  Received htlcs:
  SIGNED
REMOTE COMMITS:
 Commit 0:
  Offered htlcs:
  Received htlcs:
  Pending unacked: FEE
  SIGNED
 Commit 1:
  Offered htlcs:
  Received htlcs:
  Pending unacked: FEE
  SIGNED
 Commit 2:
  Offered htlcs:
  Received htlcs:
  SIGNED
***B after receiving fee change***
LOCAL COMMITS:
 Commit 0:
  Offered htlcs:
  Received htlcs:
  Pending unacked: FEE
  SIGNED
REMOTE COMMITS:
 Commit 0:
  Offered htlcs:
  Received htlcs:
  SIGNED
***B after receiving first commit***
LOCAL COMMITS:
 Commit 1:
  Offered htlcs:
  Received htlcs:
  SIGNED
REMOTE COMMITS:
 Commit 0:
  Offered htlcs:
  Received htlcs:
  Pending acked
  SIGNED
***B after receiving second commit***
LOCAL COMMITS:
 Commit 2:
  Offered htlcs:
  Received htlcs:
  SIGNED
REMOTE COMMITS:
 Commit 0:
  Offered htlcs:
  Received htlcs:
  Pending acked
  SIGNED
***B after committing***
LOCAL COMMITS:
 Commit 2:
  Offered htlcs:
  Received htlcs:
  SIGNED
REMOTE COMMITS:
 Commit 1:
  Offered htlcs:
  Received htlcs:
  Fee level 2
  SIGNED
